Tools/autotest/param_metadata/htmlemit.py

```diff
@@ -1,17 +1,20 @@
 #!/usr/bin/env python
+"""
+Emit docs in a form acceptable to the APM wordpress docs site
```

Comment:
On Fri, 29 Jul 2016, Francisco Ferreira wrote:

> In Tools/autotest/param_metadata/htmlemit.py:
> 
> > @@ -1,17 +1,20 @@
> >  #!/usr/bin/env python
> > +"""
> > +Emit docs in a form acceptable to the APM wordpress docs site
> 
> @peterbarker Is this still true?

I do not believe so.  I believe this was replaced with the current 
ardupilot "wiki", which uses the rstemit.py output.